spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/actuator/loggers.adoc

```diff
@@ -31,3 +31,16 @@ To configure a given logger, `POST` a partial entity to the resource's URI, as t
 ----
 
 TIP: To "`reset`" the specific level of the logger (and use the default configuration instead), you can pass a value of `null` as the `configuredLevel`.
+
+== OpenTelemetry
```

Comment:
We should ideally have 3 lines before the header and an anchor `[[actuator.loggers.opentelemetry]]`
`